### DISCUSSION TITLE
It's possible to tunnel all traffic of ios over ssh with this app?

### DISCUSSION THREAD

**Discussion by rikhtehgaran** [isontheline/pro.webssh.net#741] [Q&A]
Hi 
I have a ubuntu server somewhere (suppose Germany) and i want to tunnel all traffic on my iphone into this server I mean when I don't connect to server and I open browser and check my IP address it show's current IP connection but when I enable ssh tunnel then check my IP it shows IP of my germany server. it can be done in all of platform like windows , macOS , android but I can't find any way to do it on Iphone and the command on linux or macos for tunneling traffic is:
`ssh -o port=22 mrg@<GermanyServerIp> -D <LocalSocksPort> -C -v`

**Comment by isontheline** [comment]
Hi @rikhtehgaran 👋

Thank you so much for your feedback 🙏

It will possible when https://github.com/isontheline/pro.webssh.net/issues/607 will be done.

Please upvote #607 in order to drive the priorities.

Have a nice day ☀️

**Reply by rikhtehgaran** [reply]
Hi Man
Thanks for your reply, but as maybe you know, the internet connection is restricted in IRAN, and I need to connect my phone to the internet if you can do it very soon, you help the people in Iran. Today everybody talk about IRAN
Thanks Again

**Reply by isontheline** [reply]
Will you use your iDevice as a VPN gateway for a computer or will use use the VPN directly (apps on your iPhone)?
I think I'm able to do the VPN gateway in quick manner but the second needs more time.
